framework/freedomotic-core/src/test/java/com/freedomotic/rules/StatementTest.java

```diff
@@ -0,0 +1,113 @@
+package com.freedomotic.rules;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class StatementTest {
+
+  @Test
+  public void createLogicalInputValid() {
+    final Statement retval = new Statement().create(Statement.EQUALS, "retval1", Statement.AND, "100");
```

Comment:
I find the use of `retval1` slightly confusing as an attribute, it might be worth putting something relevant to the code. 

Similar occurrences below of `retval[number]` seems unnecessary when attribute can be set to any kind of string?